.github/workflows/phpunit-tests.yml

```diff
@@ -256,22 +269,37 @@ jobs:
       - name: Install WordPress
         run: npm run env:install
 
+      - name: Run slow PHPUnit tests on PHP 5.6
```

Comment:
```suggestion
      - name: Run slow PHPUnit tests on PHP ${{ matrix.php }}
```

Reply:
Using the matrix value here would allow more versions to be split off more easily going forward. I'll be backporting all of the improvements we've made to the older branches where appropriate, so we could potentially do this for PHP <= 5.5 as well.